pymeasure/display/widgets.py

```diff
@@ -229,6 +229,241 @@ def update_y_column(self, index):
         axis = self.columns_y.itemText(index)
         self.plot_frame.change_y_axis(axis)
 
+# JM: #########################################################################
```

Comment:
This comment should be removed before merge.

Reply:
Yup. As a rule, any comment with a JM at the beginning I was using to make development easier and is intended to be removed in a final version. I'll begin removing some of them now.